_includes/unsupported-version-soon.md

```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_danger}}
-Cockroach Labs will stop providing <strong>Assistance Support</strong> for this version on <strong>May 12, 2021</strong>. Prior to that date, upgrade to a more recent version to continue receiving support. For more details, see the <a href="https://www.cockroachlabs.com/docs/releases/release-support-policy.html">Release Support Policy</a>.
+Cockroach Labs will stop providing <strong>Assistance Support</strong> for this version on <strong>May 12, 2021</strong>. Prior to that date, upgrade to a more recent version to continue receiving support. For more details, see the <a href="release-support-policy.html">Release Support Policy</a>.
```

Comment:
@taroface This does appear to have resolved the issue. But why not markdown? E.g.`[Release Support Policy](release-support-policy.md)` Is it just a matter of choice; no effect on the output and clearly our CI isn't forcing us to avoid manual a hrefs?

Reply:
TFTR! I tried Markdown, which our callouts normally support. But it didn't render correctly on this one. I think it's because it is being displayed programmatically for the affected versions.